desktop/plugins/network/MockResponseDialog.tsx

```diff
@@ -51,6 +55,31 @@ export function MockResponseDialog(props: Props) {
         />
       </Layout.Horizontal>
       <Layout.Horizontal>
```

Comment:
Oh, another nit, could you add `gap` to this `Layout.Horzontal` to give the buttons some spacing?

Reply:
Completed.

![image](https://user-images.githubusercontent.com/337874/105764515-b0828980-5f1c-11eb-9122-fa6ac0598618.png)